src/Microsoft.AspNetCore.SignalR.Client/HttpConnectionFactory.cs

```diff
@@ -31,10 +32,10 @@ public HttpConnectionFactory(IOptions<HttpConnectionOptions> options, ILoggerFac
             _loggerFactory = loggerFactory;
         }
 
-        public async Task<ConnectionContext> ConnectAsync(TransferFormat transferFormat)
+        public async Task<ConnectionContext> ConnectAsync(TransferFormat transferFormat, CancellationToken cancellationToken)
```

Comment:
No default? Add a comment explaining why.

Reply:
just missed it